packages/@ember/octane-addon-blueprint/files/package.json

```diff
@@ -28,7 +28,7 @@
     "@ember/edition-utils": "^1.1.1",
     "@ember/optional-features": "^1.0.0",
     "@glimmer/component": "^0.14.0-alpha.12",
```

Comment:
btw `@glimmer/component` is now at [`1.0.0-beta.3`](https://www.npmjs.com/package/@glimmer/component?activeTab=versions). Was this intentionally left out?

Reply:
Whoops. I just did yarn upgrade latest interactive

Reply:
I forgot that glimmer componet was still in beta, and yarn upgrade only does real releases